tox.ini

```diff
@@ -63,5 +63,5 @@ commands=
     make travis-assets
     python setup.py sdist
     make docs
-    make mo-all
+    python setup.py build_mo
```

Comment:
Because I wanted to see if in future we can actually use `--check` here.  `make mo` is just a wrapper and one I've eliminated from the release docs now anyway.